src/hooks/useOutsideClickHandler.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { fireEvent, render, renderHook } from 'test'
+
+import { useOutsideClickHandler } from './useOutsideClickHandler'
+
+describe('useOutsideClickHandler', () => {
+  it('should call onOutsideClick when clicked outside of node', () => {
```

Comment:
Move shared lines to a `beforeEach`.

```suggestion
  let node: HTMLDivElement
  beforeEach(() => {
    node = render(<div />)
    renderHook(() => useOutsideClickHandler(node.container as HTMLDivElement, onOutsideClick))
  })

  it('should call onOutsideClick when clicked outside of node', () => {
```